src/lib/stream.rs

```diff
@@ -0,0 +1,51 @@
+use crate::prelude::{ProtonClient, ProtonClientError, Result};
+use clickhouse::query::RowCursor;
+use clickhouse::Row;
+use serde::Deserialize;
+
+impl ProtonClient {
+    ///
+    /// Executes a streaming query, returning a [`RowCursor`] to obtain results
+    /// as they become available from the stream. The key difference compared to fetch is that,
+    /// for streaming query, the returned result is a unbounded stream. Also,
+    /// a fetch_stream query will keep running continuously returning fresh data
+    /// until the application terminates..
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    ///  use proton_client::ProtonClient;
+    ///  use proton_client::prelude::Result;
+    ///
+    ///  async fn example() -> Result<()> {
+    ///
+    /// #[derive(Debug, clickhouse::Row, serde::Deserialize)]
+    /// struct MyRow {
+    ///     no: u32,
+    ///     name: String,
+    /// }
+    ///
+    /// let client = ProtonClient::new("http://localhost:3128");
```

Comment:
```suggestion
    /// let client = ProtonClient::new("http://localhost:3218");
```

Reply:
Good catch. Change accepted. @jovezhong  Feel free to merge & release.